### PR TITLE
Small collection of fixes, mostly grammer or consistency

### PR DESCRIPTION
**Pull request by ktec**
Hello,

This is just a few little fixes I made as I was reading through the library trying to learn. Feel free to pick and choose if you don't like my suggestions it's totally cool.

best regards

### CODE REVIEW

File: lib/quark/compose.ex [hunk 1]
```diff
@@ -73,7 +73,7 @@ defmodule Quark.Compose do
   def g <|> f, do: compose(g, f)
 
   @doc ~S"""
-  Function composition, from the back of the lift to the front
+  Function composition, from the back of the list to the front
```
Comment (OvermindDL1) [inline lib/quark/compose.ex]: I still think this one seems unclear, maybe it should instead be:
```
Function composition, pipes a value to the first argument then that output is piped to the second argument.
```
Or something like that?

Comment (ktec) [inline lib/quark/compose.ex]: I was trying to go for small changes. I feel like introducing so many words like "pipes" and "arguments", "first" and "second" etc add confusion to the sentence.
```
Function composition, pipes a value to the first argument then that output is piped to the second argument.
```
Perhaps we're trying to say:
```
Function composition, from left to right
```
Assuming some basic knowledge of what it means to compose two functions, I would have thought most people would understand this?

Comment (ktec) [inline lib/quark/compose.ex]: Or alternatively, perhaps this:
```
compose # Function composition, from the tail of the list to the head
compose_forward # Function composition, from the head of the list to the tail
```

Comment (OvermindDL1) [inline lib/quark/compose.ex]: Except there is no list, they just take the arguments straight.  That is what is confusing to me.  :-)
It was probably a mis-copy/paste or so originally?  All it does is turn:
```elixir
sum_plus_one = compose_forward(&(Enum.sum(&1)), &(&1 + 1))
[1,2,3] |> sum_plus_one.()
```
To basically just become this:
```elixir
[1,2,3]
|> (&(Enum.sum(&1)).()
|> (&(&1 + 1)).()
```
The list is only used in the example, it is not part of the definition and does not belong in the definition's description.

Comment (ktec) [inline lib/quark/compose.ex]: Ah ok yes, so perhaps "lift" was not a typo after all. I've not heard the term used in that context, but that doesn't mean its not a valid statement.

Comment (OvermindDL1) [inline lib/quark/compose.ex]: 'lift' does not really seem proper in this context to me either, I really think it was a mis-copy/paste?  :-/

Comment (expede) [inline lib/quark/compose.ex]: Oh, yes, there's a variant on lists. This is totally a copy/paste and _then_ a typo 😅 Thanks for catching these!

File: mix.exs [hunk 2]
```diff
@@ -22,7 +22,7 @@ defmodule Quark.Mixfile do
       aliases: ["quality": ["test", "credo --strict"]],
 
       deps: [
-        {:credo,    "~> 0.4",  only: [:dev, :test]},
+        {:credo,    "~> 0.4.14",  only: [:dev, :test]},
```
Comment (expede) [inline mix.exs]: Why pin Credo to the patch?

**Comment by expede** [comment]
Neeeevermind, I see the comment now

> 	Update Credo  …
> A bug caused by credo relying on a Hex Util that no longer exists, is
> detailed here:
> 
> rrrene/credo#210
>
> This commit just updates the minor version to include the patch.

**Comment by ktec** [comment]
Sorry I missed this earlier, crazy week! 
Yay \o/ to making OS authors happy! 👩‍🚀

**Pull request merged**